bittensor/extrinsics/unstaking.py

```diff
@@ -211,7 +208,7 @@ async def unstake_extrinsic(
             )
             return False
 
-    except bittensor.errors.NotRegisteredError:
+    except bittensor.errors.NotRegisteredError as e:
```

Comment:
Are we doing anything with this `e` var?

Reply:
Nope